recipes/gifrop/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "gifrop" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "7059c916b2deec451a97037936bc4905c44211d12a87d9db3e8724a9797672f0" %}
+{% set user = "jtrachsel" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/{{ user }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+ host:
+  - abricate=1.0.0
+  - parallel
+  - roary=3.13.0
+  - prokka=1.14.6
+  - r-base=3.6.3
```

Comment:
Does it have to be exactly 3.6.3? Is there any reason the 4.0 won't be supported?

Reply:
I changed these to be the minimum version requirements